pom.xml

```diff
@@ -71,12 +71,14 @@
         <buildNumber>${user.name}-${maven.build.timestamp}</buildNumber>
         <project.build.outputTimestamp>2022-03-22T23:08:15Z</project.build.outputTimestamp>
         <jacoco.skip>true</jacoco.skip>
+        <surefire.version>3.0.0-M7</surefire.version>
         <!--suppress CheckTagEmptyBody -->
         <surefire.argLine></surefire.argLine>
         <!--suppress CheckTagEmptyBody -->
         <failsafe.argLine></failsafe.argLine>
         <nexus.deploy.skip>false</nexus.deploy.skip>
-
+        <jacocoPort>52212</jacocoPort>
```

Comment:
Is it possible to use a random port? We can have some conflicts on CI platform.

Reply:
Usually, it is not needed to specify a port. Even with integration tests. At least when using meecrowave, as it sits in the same JVM.

Reply:
Actually, this is no longer necessary